js/index.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+	$('form').submit(addEntry);
+
+	function addEntry (event) {
+		event.preventDefault();
+
+	var input = $('#city-type').val();
+
+		if(input === 'San Fransisco') {
```

Comment:
How would you handle `San Francisco` OR `SF`? or `Bay Area`?

If you look at the rubric, it asks you to handle multiple conditions for each city.  Take a look at this: https://github.com/wjpurdum/FEWD22-Assignment6#technical-requirements. Let me know if you have questions!